### PR TITLE
fix(dependency): cleanup and upgrade some

### PR DESCRIPTION
**Pull request by bodinsamuel**
## Describe your changes

Fixes NAN-1129

- Upgraded vulnerable aws-sdk
- Removed all unused packages 
This should fasten local and CLI install
- Upgraded minor stuff in the UI
- Use `file:..` for types 
not sure why I used the version

### CODE REVIEW

File: packages/webapp/src/pages/AccountSettings.tsx [hunk 1]
```diff
@@ -140,10 +140,10 @@ export default function AccountSettings() {
                 <Modal.Content>
                     <p>This action cannot be undone, are you sure?</p>
                 </Modal.Content>
-                <Modal.Action placeholder={null} passive onClick={() => cancelSuspendMember()}>
+                <Modal.Action placeholder={null} passive onClick={() => cancelSuspendMember()} onPointerEnterCapture={null} onPointerLeaveCapture={null}>
```
Comment (bodinsamuel) [inline packages/webapp/src/pages/AccountSettings.tsx:143]: bug in geist ui https://github.com/geist-org/geist-ui/issues/858

**Review by khaliqgant — APPROVED** [review]
nice, thanks

**Comment by bodinsamuel** [comment]
Just as I merged master, there is now an issue with rollup optional dep 😮‍💨

**Comment by khaliqgant** [comment]
> Just as I merged master, there is now an issue with rollup optional dep 😮‍💨

Ugh 😩

**Comment by bodinsamuel** [comment]
Seems better now 🙌🏻

**Pull request merged**